lib/control/validation/token.js

```diff
@@ -1,52 +1,35 @@
 'use strict';
 
-const HTTP = require('http');
 const bad = require('../validation/badRequest');
-
-function send(body, ami, vault, callback) {
-
-  const request = new Buffer(JSON.stringify(body, null, 2));
-
-  const req = HTTP.request({
-    hostname: vault.hostname,
-    port: vault.port,
-    method: 'POST',
-    path: '/v1/auth/token/create',
-    headers: {
-      'X-Vault-Token': vault.token,
-      'content-length': request.length
+const vaultRequest = require('../util').vaultRequest;
+
+function getTokenTiming(accessor, vault) {
+  let creation_time = null,
+      explicit_max_ttl = null;
+
+  return vaultRequest(vault, `/v1/auth/token/lookup-accessor/${accessor}`, 'POST').then((resp) => {
+    creation_time = resp.data.creation_time * 1000;
+    explicit_max_ttl = resp.data.explicit_max_ttl * 1000;
+  }).then(() => {
+    if (explicit_max_ttl === 0) {
```

Comment:
Is there a case where `explicit_max_ttl` is `null` or `NaN` or less than zero when we get here?